vault/changelog.release.2022-08-09.md

```diff
@@ -0,0 +1,71 @@
+---
+id: 482bl67lvjd0r8ht0cngxmk
+title: 0.107
+desc: ''
+updated: 1660015273752
+created: 1659968708044
+---
+
+Dendron 0.107 has sprouted  🌱
+August 09, 2022
+
+**Preview Locking**: You can now lock the preview to a specific note which will remain pinned even as you navigate to different notes.
+
+- More information: [[Toggle Preview Lock|dendron://dendron.dendron-site/dendron.topic.preview.commands.toggle-preview-lock]]
+
+<div style="position: relative; padding-bottom: 64.5933014354067%; height: 0;"><iframe src="https://www.loom.com/embed/6e35934a0c534f3585e750e5eef2e739" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Highlights
+
+- feat(view): add "Toggle PreviewLock" command (#3293) @sam ([[docs|dendron.topic.preview.commands.toggle-preview-lock]])
+
+## Everything Else
+
+- enhance(structure): support xvault template in note traits ([[docs|dendron://dendron.dendron-site/dendron.topic.traits.api#return]])
+- enhance(publish): speed up published site
+- enhance(views): toggle preview lock from inside preview ([[docs|dendron.topic.preview.commands.toggle-preview-lock]])
+- enhance(workspace): only show the feature showcase once 
+- enhance(lookup): add space between note title and vault name
+- fix(views): nested bullets formatted correctly in preview
+- fix(lookup): validate file name on note lookup ([[docs|dendron.topic.lookup.ref.restrictions]])
+- fix(workspace): remote vault not recognized by dendron for windows
+- fix(workspace): always treat `begin|end` block anchors as valid
```

Comment:
see https://github.com/dendronhq/dendron-site/pull/608/files#r940903144 
This should be remove (will be in next release)